lib/jekyll/rp_logs/rp_logline.rb

```diff
@@ -145,7 +145,13 @@ def mergeable_with?(next_line)
       end
 
       def merge!(next_line)
-        @contents += "#{space_between_lines}#{next_line.contents}"
+        if /\.\.\.\w*$/.match(@contents) && /^\w*\.\.\./.match(next_line.contents)
```

Comment:
Why are you testing for `...\w+` on the first line and `\w+...` on the next line? That seems backwards...?